app/grants/sync/binance.py

```diff
@@ -1,6 +1,9 @@
+import logging
+
 import requests
 from grants.sync.helpers import record_contribution_activity
 
+logger = logging.getLogger(__name__)
```

Comment:
do we need this ?

Reply:
yea... it was already being used in L45 but not declared (bug)